### ISSUE TITLE
Report unnecessary exports

### ISSUE THREAD

**Issue by ocharles** [ocharles/weeder#59]
I think it's in `weeder`'s territory to report exports that actually aren't needed.

**Comment by DamienCassou** [comment]
I agree. I have a package that is divided into a lib and an executable. I would like to remove everything that is exported by the lib but unused by the executable. Weeder won't detect functions that are exported by exposed modules but unused.
